cli/module_registry.go

```diff
@@ -144,6 +144,10 @@ func UpdateModuleAction(c *cli.Context) error {
 		}
 	}
 
+	if manifest.Visibility != moduleVisibilityPrivate && manifest.Visibility != moduleVisibilityPublic {
+		return errors.Errorf("manifest visibility must be either 'private' or 'public' but you provided %s", manifest.Visibility)
```

Comment:
if this is never changed, it will be an empty string which will fail @Fahmina

Reply:
I think that this error case is already handled.. not bad to check again though
```
➜  cli git:(zp/2488) ✗ ./main module update
Error: invalid module visibility. must be either "public" or "private"
```